src/app/components/top-bar/top-bar.scss

```diff
@@ -23,14 +23,18 @@
     }
     .title {
         width: 100%;
-        text-align: left;
         color: grey;
+        padding-left: 0px;
     }
 
     .center {
         text-align: center;
     }
 
+    .middle-padding {
+        padding-right: 56px;
```

Comment:
why is this needed ? on small screens it occupies 56px on the right, and if the page title is long like (Recover wallet) `...` will be shown event if there is enough space.

Reply:
This is equivalent with the space required for the "X"  . I added this so I can use center text for the middle part even when we dont have a right section on the top bar. On my 320px × 568px setup in chrome i didnt had problems with the display. I can delete this but then, for a better fixing of the middle section I think I need to remove the center text from it also when there is no right section. I noticed that the center text makes that middle section not alligned. What do you think is best Tibi ?

Reply:
with the latest changes i think it's much better, for now. We can do improvements in future...